rocAL/rocAL_pybind/examples/tf_petsTrainingExample/requirements.sh

```diff
@@ -0,0 +1,8 @@
+# installing required packages
+pip install contextlib2
+git clone https://github.com/tensorflow/models.git
+cd models/research
+protoc object_detection/protos/*.proto --python_out=.
+cp object_detection/packages/tf2/setup.py .
+python -m pip install --use-feature=2020-resolver .
```

Comment:
@swetha097 The flag "2020-resolver" was not working when I tried previously. The install worked with removing the flag altogether. ``` option use-feature: invalid choice: '2020-resolver'```
Is that not the situation anymore?
Also, if you notice all these are installed in the [tensorflow docker](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/0a4e22ab837ea4e0efcf215fe1d99cae36c6df7b/docker/tensorflow/mivisionx-with-tensorflow.dockerfile#L55). Why do we need a separate script again?

Reply:
Yes Lakshmi, we dont see a issue with this now in pytorch docker - `option use-feature: invalid choice: '2020-resolver'`

We have mentioned in the README that the requirements is only for pytorch docker & not for tensorflow docker

Reply:
@swetha097 Does the tf docker need the flag or not? If it doesn't please change the dockerfile also

Reply:
```
RUN cd ~/models/research/ && \
protoc object_detection/protos/*.proto --python_out=. && \
cp object_detection/packages/tf2/setup.py . && \
pip3 install --use-feature=2020-resolver .
```

This is the CMD used in tf docker, its same as what we gave for pytorch here.

Reply:
This flag is used there also.